tests/it/homeassistant_HassTurnOff.yaml

```diff
@@ -1,66 +1,66 @@
-language: it
+language: "it"
 tests:
   - sentences:
-      - spegni l'interruttore cucina
-      - spegni interruttore cucina
-      - puoi spegnere l'interruttore cucina
-      - potresti disattivare l'interruttore cucina per favore
-      - per favore spegni interruttore cucina
-      - disattiva interruttore cucina grazie
+      - "spegni l'interruttore cucina"
+      - "spegni interruttore cucina"
+      - "puoi spegnere l'interruttore cucina"
+      - "potresti disattivare l'interruttore cucina per favore"
+      - "per favore spegni interruttore cucina"
+      - "disattiva interruttore cucina grazie"
     intent:
-      name: HassTurnOff
+      name: "HassTurnOff"
       slots:
-        name: switch.cucina
+        name: "switch.cucina"
   - sentences:
-      - chiudi la tenda sinistra
+      - "chiudi la tenda sinistra"
     intent:
-      name: HassTurnOff
+      name: "HassTurnOff"
       slots:
-        name: cover.tenda_sinistra
+        name: "cover.tenda_sinistra"
   - sentences:
-      - chiudi la tenda sinistra del soggiorno
-      - abbassa la tenda sinistra del soggiorno
+      - "chiudi la tenda sinistra del soggiorno"
+      - "abbassa la tenda sinistra del soggiorno"
     intent:
-      name: HassTurnOff
+      name: "HassTurnOff"
       slots:
-        name: cover.tenda_sinistra
-        area: soggiorno
+        name: "cover.tenda_sinistra"
+        area: "soggiorno"
```

Comment:
This should not be quoted as in https://developers.home-assistant.io/docs/documenting/yaml-style-guide/#strings-continued